### PR TITLE
Adapter450

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Adapter V4 Release Notes
 
+## NUnit3 Test Adapter for Visual Studio and Dotnet - Version 4.5.0 - May 30, 2023
+
+This is a version supporting the upcoming NUnit 4 version.  
+It further uses a released version of the NUnit.Engine, version 3.15.4.
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/AdapterV4-Release-Notes.md:6]:
```suggestion
This is a version supporting the upcoming NUnit version 4.  It uses a released version of the NUnit.Engine, version 3.15.4.
```

File: docs/articles/vs-test-adapter/Supported-Frameworks.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Version 4.4 of the adapter will support future versions of .net, as long as ther
 
 The included versions are as shown below, only the versions where changes are done are listed.
 
-|Adapter version|Test framework version|Test Net (core) version|Comment|
+|Adapter version|.Net framework version|.Net (core) version|Comment|
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Supported-Frameworks.md:27]:
```suggestion
| Adapter version | .NET framework version | .NET (core) version | Comment |
```

File: docs/articles/vs-test-adapter/Supported-Frameworks.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Version 4.4 of the adapter will support future versions of .net, as long as ther
...
 |---|---|---|---|
 |3.8|3.5|1.0|Some versions included 2.0 too|
 |3.16.0|3.5|2.1||
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Supported-Frameworks.md:30]:
```suggestion
| 3.8 | 3.5 | 1.0 | Some versions included 2.0 too |
| 3.16.0 | 3.5 | 2.1 | |
```

File: docs/articles/vs-test-adapter/Tips-And-Tricks.md [hunk 4]
```diff
@@ -306,6 +307,12 @@ If set, this setting will skip execution for an assembly if no tests are found d
 
 (From version 4.2.0)
 
+#### AllowParallelWithDebugger
+
+If set, this setting will allow parallel execution of tests even if the debugger is attached.  The default is false.
```
Comment (SeanKilleen) [inline docs/articles/vs-test-adapter/Tips-And-Tricks.md:312]:
```suggestion
If set, this setting will allow parallel execution of tests even if the debugger is attached.  The default is `false`.
```

**Review by SeanKilleen — COMMENTED** [review]
Minor comments but in general LGTM

**Comment by OsirisTerje** [comment]
Thanks for your comments @SeanKilleen . Always appreciate them!

**Pull request merged**